bootc/embedded-services/quadlets/nova_compute/nova_compute.yaml

```diff
@@ -0,0 +1,177 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    bind-mount-options: /var/lib/openstack/cacerts/nova/tls-ca-bundle.pem:z
+  labels:
+    app: novacompute
+  name: novacompute
+spec:
+  containers:
+  - args:
```

Comment:
This needs to be moved to an actual initContainer. Otherwise the pod will be degraded once this exits.